resources/Schema/Components/ExplosionEffect.xml

```diff
@@ -11,7 +11,9 @@
 	<RandomnessScalar>1</RandomnessScalar>
 	<Velocity X="2" Y="2"/>
 	<ColorByDistance>0</ColorByDistance>
-	<!--<ReverseAnimation>0</ReverseAnimation>-->
 	<!--<Wireframe>0</Wireframe>-->
 	<ExponentialAccelaration>0</ExponentialAccelaration>
+	<Pulsate>0</Pulsate>
```

Comment:
perhaps change "0" to "false" since this is a bool. even though ofcourse a 0 would work too